languages/purescript/exercises/concept/basics/src/Basics.purs

```diff
@@ -0,0 +1,15 @@
+module Basics where
+
+import Prelude
+
+expectedMinutesInOven :: Int
+expectedMinutesInOven = -- You need to implement this function!
+
+remainingMinutesInOven :: Int -> Int
+remainingMinutesInOven minutesInOven = -- You need to implement this function!
+
+preparationTimeInMinutes :: Int -> Int
+preparationTimeInMinutes numberOfLayers = -- You need to implement this function!
+
+totalTimeInMinutes :: Int -> Int -> Int
+totalTimeInMinutes numberOfLayers minutesInOven = -- You need to implement this function!
```

Comment:
The idea of the original `basics` exercise is to teach students how to define functions. As such, other tracks opted to put TODO's in the stub file for this exercise instead of stub function definitions to force the student to get them to write the functions themselves (which is the best way for them to learn). The obvious downside is that the code won't compile, but we feel that this is less important than getting the students to actually write the functions from scratch.